sporkfish/searcher/searcher_factory.py

```diff
@@ -25,6 +26,8 @@ def create(searcher_cfg: SearcherConfig, evaluator: Evaluator) -> Searcher:
             return NegamaxSp(evaluator, searcher_cfg)
         elif searcher_cfg.search_mode is SearchMode.LAZY_SMP:
             return NegaMaxLazySmp(evaluator, searcher_cfg)
+        elif searcher_cfg.search_mode is SearchMode.PVS:
```

Comment:
As discussed offline PVS can also be parallelized but given we don't use LAZY_SMP at all for now, for a future task.

Reply:
#143.

Reply:
Got u